ZAPD/ZRoom/Commands/SetMesh.cpp

```diff
@@ -355,7 +355,7 @@ void SetMesh::GenDListDeclarations(std::vector<uint8_t> rawData, ZDisplayList* d
 	for (pair<uint32_t, string> vtxEntry : dList->vtxDeclarations)
 	{
 		zRoom->parent->AddDeclarationArray(
-			vtxEntry.first, DeclarationAlignment::Align8,
+			vtxEntry.first, DeclarationAlignment::None,
```

Comment:
Why?

Reply:
I'm not sure, by I encountered several instances in MM where these were not aligned to 8-bytes

Reply:
You could add the same check here too.